src/widget/flex.rs

```diff
@@ -166,20 +166,25 @@ impl<T: Data> Widget<T> for Flex<T> {
         }
 
```

Comment:
Where it has `minor = 0.0f64` above, I think that should be `minor = self.direction.minor(bc.min())`. (Another nit, we should probably be calling the accessors rather than getting the fields directly, in case people want to cut'n'paste this code). I don't think this is a very serious issue, but I'd like to move toward making sure sizes actually meet the constraints by construction.